elections/kubeflow-steering-committee-elections-2023.md

```diff
@@ -1 +1,92 @@
+
+The PSC responsibilities and operations are defined [here](https://github.com/kubeflow/community/blob/master/proposals/STEERING-COMMITTEE.md). The PSC is intended to be a resource in the community that both guides the direction of the Kubeflow Project and acts as a resource to the TOC, AOC, Working Groups (WG), Special Interest Groups (SIG) and Teams. Their role is also to help define what new WGs, SIGs and Teams. 
+
+The Project Steering Committee elections process for open positions will start on **4 December 2023**. 
+
+More details about the Kubeflow election process can be found in the [Kubeflow Election Process document](https://github.com/kubeflow/community/blob/master/proposals/kubeflow-steering-committee-election-proposal.md).  
+
+
+## Voter Qualifications ##
+
+**Voter qualifications:** To be eligible to vote in this election a voter should be an active member in the Kubeflow Community and have a minimum of 50 contributions* to the Kubeflow project in the last year.**  If you believe you are eligible to vote, but are not listed as eligible you may [file an exception](https://forms.gle/epaMrirZCNBztoRz5). 
+
+*Contributions are defined as:  Github Code and Issue Authors, Merge and Pull Requests Submitters as well as commenters on all issues, PRs and MRs.
+
+** For this election last year is from 3 December 2022 to 3 December 2023
+
+## Nomination Qualifications ##
+
+**Nomination qualifications:** A nominee should be an active member in the Kubeflow Community and have a minimum of 50 contributions* to the Kubeflow project in the last year.** have the available bandwidth to be invested in the PSG, and be able to operate neutrally in discussions regardless of company or project affiliation. If you believe you are eligible, but are not you may [file an exception](https://forms.gle/epaMrirZCNBztoRz5).  *(Election officers are ineligible to run in an election.)* 
+
+**Contributions are defined as:  Github Code and Issue Authors, Merge and Pull Requests Submitters as well as commenters on all issues, PRs and MRs.
+
+** For this election last year is from 3 December 2022 to 3 December 2023
+
+## Process for nominations ##
+
+If you would like to nominate yourself or someone you know who is qualified, you can do so by adding their name to the following Nominations Issues. *(Nominations Issue will be created on the date the nominations phase will be open)*
+
+We will publish the list of Qualified Nominees.  
+
+
+## Election Timeline ##
+
+**Exceptions Phase:**  4 December 2023 at 0900 Pacific Time  (Starts) - 10 December 2023 at 12:00pm Pacific Time (Ends)
+
+**Nominations Phase:** 11 December 2023 at 0900 Pacific Time  (Starts) - 24 December 2023 at 12:00pm Pacific Time (Ends)
+
+**Testimonial Phase:** 25 December 2023 at 0900 Pacific Time (Starts) - 7 January 2024 at 12:00pm Pacific Time (Ends)
+
+**Voting Phase:** 8 January 2024 at 0900 Pacific Time (Starts) - 29 January 2024 at 12:00pm Pacific Time (Ends)
```

Comment:
could this be completed by mid January ?

Reply:
@jbottum no, as I already published the timeline this morning. And with the holidays, vacation schedules, people returning from events etc we don't want to rush these phases.